docs/cli.md

```diff
@@ -523,10 +523,33 @@ gcli role list
 
 ```bash
 gcli role create --role admin
- ```
+```
 
 #### Delete a role
 
 ```bash
 gcli role delete --role admin
- ```
+```
+
+#### Add a role to a user
+
+```bash
+gcli user update --metalake metalake_demo --user new_user --role admin
```

Comment:
Can we use "grant" / "revoke" as the command to operate the role to user&groups? "update"/"delete" is not that accurate for this kind of operations.

Reply:
Sure I went this was as Jerry prefered a "less is more approach"